.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
:bulb: This (`trim_trailing_whitespace`) is likely to cause churn in the code base over time. I would leave it out but it could go either way.